activerecord/CHANGELOG.md

```diff
@@ -17,7 +17,7 @@
     config.active_record.raise_on_assign_to_attr_readonly = true
     ```
 
-    and will be enabled by default with `load_defaults 7.1`.
+    and will be enabled by default with `load_defaults("7.1")`.
```

Comment:
I think Rails generates the load_defaults version with a Float and without parantheses:
https://github.com/rails/rails/blob/fb1ab3460a676ce7def0819c2e92289ef2dcbe3b/railties/lib/rails/generators/rails/app/templates/config/application.rb.tt#L31

```suggestion
    and will be enabled by default with `config.load_defaults 7.1`.
```

Reply:
Huh, I didn't notice that. I guess I assumed a string because it's treated as a string [here](https://github.com/rails/rails/blob/eb303ad1cf7d5e001b71508bbd140510e2ac3bb6/railties/lib/rails/application/configuration.rb#L112).

Reply:
Yes, that was suprising to me as well 😄